authenticator/server/config/config.go

```diff
@@ -1,31 +1,24 @@
-package server
+package config
 
 import "github.com/kelseyhightower/envconfig"
 
 // Config is an object for storing configuration variables set through
-// Approzium's environment. Supports:
-// 		- APPROZIUM_HOST: Defaults to 127.0.0.1.
-//		- APPROZIUM_PORT: Defaults to 6000.
-//		- APPROZIUM_LOG_LEVEL: Defaults to info. Valid values are:
-//			- trace
-//			- debug
-//			- info
-//			- warn
-//			- error
-//			- fatal
-//			- panic
+// Approzium's environment.
```

Comment:
TODO link to docs for more.

Reply:
https://github.com/approzium/approzium/pull/95/commits/3c77fddd8e11e7db3403961be4b5367f3c682a95